NSeleneTests/Integration/SharedDriver/Harness/Constraints/TimeoutConstraint.cs

```diff
@@ -0,0 +1,39 @@
+﻿using NUnit.Framework.Constraints;
+
+namespace NSelene.Tests.Integration.SharedDriver.Harness.Constraints
+{
+    internal class TimeoutConstraint(string errorMessage) : Constraint
+    {
+        private readonly string errorMessage = $$"""
+            Timed out after {{Configuration.Timeout}}s, while waiting for:
```

Comment:
Heh:) Tightly coupled to the "global variable" that is also might be not obvious from the usage:) 
the accuracyDelta below is also kind of "hardcoded"... I would at least inject these values as dependencies, then use same defaults, but at least having an option to parametrize if needed... 
But OK, we can keep it as it is for now... and later I can rafactor these things for this constraint...